setup.py

```diff
@@ -44,6 +44,10 @@ def check_torch_version():
             "prometheus_client",
             "parallel-ssh",
             "pydantic",
+            "setuptools_rust",
+            "tqdm",
+            "colorama",
+            "requests",
```

Comment:
don't add these deps to bagua, rust related deps are already in `bagua-core`

Reply:
OK, Useless dependencies have been removed. `requests` need to stay.